src/ucp/wireup/address.c

```diff
@@ -539,6 +542,11 @@ ucp_address_packed_size(ucp_worker_h worker,
         }
     }
     return size;
+
+err:
+    ucs_debug("failed to pack address for device %s",
+              worker->context->tl_rscs[dev->rsc_index].tl_rsc.dev_name);
```

Comment:
IMO better have single failure message, by passing the device name to ucp_address_packed_value_size

Reply:
fixed